lib/sprig/helpers.rb

```diff
@@ -5,9 +5,12 @@ def seed_directory
     end
 
     def sprig(directive_definitions)
-      hopper = []
-      DirectiveList.new(directive_definitions).add_seeds_to_hopper(hopper)
-      Planter.new(hopper).sprig
+      plant_records(directive_definitions)
+    end
+
+    def sprig_shared(directive_definitions)
+      Sprig.configuration.seeds_directory = 'shared'
```

Comment:
Yeah, I was not sure about this specific line, but I found it as clearest way to do it without modifying the whole architecture.

Any ideas to deal with the helper?
```
    def seed_directory
      Sprig.configuration.directory
    end
```